### DISCUSSION TITLE
Zero viscosity with LES

### DISCUSSION THREAD

**Discussion by makrandak** [Nek5000/nekRS#582] [Q&A]
Hello, 

I am doing a numerical experiment, where I want to simulate infinite Re number but with regularization (LES). The hypothesis is that the LES will provide enough model viscosity to drain the energy from the small scales and provide stabilization. 

When I try this in a periodic box with a homogeneous isotropic turbulence setup. I get an error from `pcg` that the beta is zero. My speculation is that the LES regularization settings are not providing enough regularity to make the matrix inversion stable from the IMEX scheme.  

Would love to hear your thoughts on this setup.  If there is a suggestion to do this without any matrix solves those are welcome as well.  

Thanks for your help and time. 

With sincere regards,
Makrand

**Comment by stgeke** [comment]
Could you clarify the value you're using for viscosity?
I'm concerned that our current regulation schemes may not adequately address the use case you're aiming for.

**Reply by makrandak** [reply]
Hello, 

I am just trying to use zero right now, I have also tried really small values like 1e-12, and 1e-10. 
The idea is that the regularization should provide some "LES model" viscosity depending on the mesh (polynomial order) to make the system solvable. 
But if the regularization is done after the solve, the solver would run into problems, I am speculating that is what is happening right now.

**Reply by stgeke** [reply]
The hpfRT regularization incorporates a diffusion-like term added to the RHS. You can experiment with the nModes and scalingCoeff parameters of hpfRT, but it's unlikely that these adjustments will achieve the desired level of diffusivity.

**Reply by stgeke** [reply]
I suspect that avm might work, but it is typically too diffusive. Additionally, it is currently not available for velocity applications (for no good reason).

**Reply by makrandak** [reply]
I will look into all your suggestions. Thanks for them. 

Is there a plan to add AVM for velocity in future releases? in the `next` branch? I am not saying it will be a solution for what I am trying, but it could be something I could try naturally.
